packages/snaps-sdk/src/types/handlers/user-input.ts

```diff
@@ -71,11 +72,31 @@ export const FileStruct = object({
  */
 export type File = Infer<typeof FileStruct>;
 
+export const AssetSelectorStateStruct = object({
+  asset: CaipAssetTypeStruct,
+  name: string(),
+  symbol: string(),
+});
+
+/**
+ * The state of the asset selector component.
+ *
+ * @property asset - The CAIP-19 asset ID.
+ * @property name - The name of the asset.
+ * @property ticker - The ticker of the asset.
```

Comment:
Do we wanna use ticker or symbol? Looks like the struct uses symbol but the docs use ticker

Reply:
symbol seems to be the standard, I forgot to update the doc thanks